src/UI/Text.elm

```diff
@@ -14,251 +15,100 @@ module UI.Text exposing
     , subtitle1
     , subtitle2
     , toEl
+    , withColor
     )
 
 import Element exposing (Attribute, Element)
 import Element.Font as Font
 import List
+import UI.Internal.Text as Internal exposing (..)
```

Comment:
what do you think about not expose all here? I got confused a little to identify where a function comes from e.g `attributes`